profiles/dini.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+// In case everything else fails, we want the error
+window.addEventListener("error", ev => {
+  console.error(ev.error, ev.message, ev);
+});
+
+const modules = [
+  // order is significant
+  import("../src/core/base-runner.js"),
+  import("../src/core/ui.js"),
+  import("../src/core/location-hash.js"),
+  import("../src/core/l10n.js"),
+  import("../src/dini/defaults.js"),
+  import("../src/core/style.js"),
+  import("../src/w3c/style.js"),
```

Comment:
You probably don't want to include this one. The W3C could change styles at any point and this might break things.
 
```suggestion
```

Reply:
I copied `style.js` for now, but kept the links to the W3C stylesheets since we are using them at DINI, too. I am aware that this may break anytime, if the W3C changes their stylesheets. But for now we don't have our own stylesheets for DINI.

https://github.com/w3c/respec/pull/2739/commits/2aa34468932b8614e96ebc53a4760ab3bba9f8a9